v4/source/users.yaml

```diff
@@ -754,6 +754,38 @@
             if ($resp->getStatusCode() == 200) {
                 $users = json_decode($resp->getBody());
             }
+  /users/known:
+    get:
+      tags:
+        - users
+      summary: Get user ids of known users
+      description: |
+        Get the list of user ids of users with any direct relationship with a
+        user. That means any user sharing any channel, including direct and
+        group channels.
+        ##### Permissions
+        Must be authenticated.
+
+        __Minimum server version__: 5.23
+      responses:
+        "200":
+          description: Known users ids retrieval successful
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UsersStats"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+      x-code-samples:
+        - lang: Go
+          source: |
+            import "github.com/mattermost/mattermost-server/model"
```

Comment:
Actually created https://mattermost.atlassian.net/browse/MM-24438 to address this. `v5` should be used in the import path.

Reply:
Very good point